packages/origin-backend/src/pods/user/dto/user.dto.ts

```diff
@@ -32,7 +33,7 @@ export class UserDTO implements IUser {
     lastName: string;
 
     @ApiProperty({ type: String })
-    @IsNotEmpty()
+    @IsEmail()
     @IsString()
```

Comment:
Is removing `@IsNotEmpty()` intentional? Or did you want to replace `@IsString` with `@IsEmail`?

Reply:
In face, having `IsEmail` should be enough.

Reply:
We also don't need IsString if we have IsEmail